sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/msfaa-integration/_tests_/msfaa-full-time-process-response-integration.scheduler.e2e-spec.ts

```diff
@@ -229,5 +238,59 @@ describe(
         currentMSFAA.id,
       );
     });
+
+    it("Should cancel a pending MSFAA record when the same MSFAA record is received in the response file and persist a MSFAA cancellation notification for the same.", async () => {
+      // Arrange
+      // Create a MSFAA record in pending state with the same MSFAA number as in the msfaa-full-time-receive-file-with-single-cancellation-record.dat
+      const student = await saveFakeStudent(db.dataSource);
+      const studentUserId = student.user.id;
+      const notificationMessageType = NotificationMessageType.MSFAACancellation;
+      await db.msfaaNumber.save(
+        createFakeMSFAANumber(
+          { student },
+          {
+            msfaaState: MSFAAStates.Pending,
```

Comment:
createFakeMSFAANumber created with default state pending right?

Reply:
Yes. Removed it @dheepak-aot . Thanks 👍